Obsidian.API/_Types/Inventory/ItemMetaBuilder.cs

```diff
@@ -28,11 +28,11 @@ public class ItemMetaBuilder
 
         private readonly List<ChatMessage> lore = new List<ChatMessage>();
 
-        public ItemMetaBuilder()
+        public ItemMetaBuilder(string name = "")
         {
             this.Enchantments = new ReadOnlyDictionary<EnchantmentType, Enchantment>(this.enchantments);
             this.StoredEnchantments = new ReadOnlyDictionary<EnchantmentType, Enchantment>(this.storedEnchantments);
-
+            this.Name = name;
```

Comment:
Not sure about passing down an empty string as default argument to a property.

Reply:
I have removed the constructor and left it as-is for now, the warning will be back for it though.